### PR TITLE
Import CEV-Eris maintenance doors

### LINKED ISSUE #950: Port Eris maintenance airlock sprites.
Gonna need all of these for Saltern so if anybody's up for this that'd be appreciated.

### PR DESCRIPTION
**Pull request by Hugal31**
This took me a lot more time I was thinking. I sympathize with PJB for doing the other doors by hand.

Here is the script I used to make the import, for posterity: [doorimporter.zip](https://github.com/space-wizards/space-station-14/files/4673366/doorimporter.zip).

Some remarks: 
- The unlit_open sprite is empty, because the leds of the maintenance doors (at least on CEV-Eris) are off when the door is not moving.
- The lights are overflowing on the rest of the door. I choosed to include the pixels around the leds to keep this effect in the unlit sprite. Go [check them](https://raw.githubusercontent.com/Hugal31/space-station-14/0a02eea239199ec7ec86488800af4d5b336d11f3/Resources/Textures/Buildings/Doors/airlock_maint_engi.rsi/opening_unlit.png) if I was not clear.
- We have to explain the "T" on the security maintenance doors.

Closes #950

doors

### CODE REVIEW

**Comment by PJB3005** [comment]
> We have to explain the "T" on the security maintenance doors.

Probably best to make these red and change that T to an S, so they aren't ironhammer anymore. I'd recommend a hue shift for it.

**Comment by PJB3005** [comment]
> The lights are overflowing on the rest of the door. I choosed to include the pixels around the leds to keep this effect in the unlit sprite. Go check them if I was not clear.

I would try to manually create a version of this that doesn't have any of the background pixels and is semitransparent on the glow, then re-use it for all the airlocks.

**Comment by Hugal31** [comment]
> I would try to manually create a version of this that doesn't have any of the background pixels and is semitransparent on the glow, then re-use it for all the airlocks.

That's what I tried first, but in the dark we only see the green, not the door behind.

> Probably best to make these red and change that T to an S, so they aren't ironhammer anymore. I'd recommend a hue shift for it.

I am not very confident with my art skills. I'll try, but if someone as better gimp-fu than be, you are welcome to try.

**Comment by PJB3005** [comment]
> I am not very confident with my art skills. I'll try, but if someone as better gimp-fu than be, you are welcome to try.

Just do a HSV adjust on it, I did the same for the security airlocks.

**Comment by SweptWasTaken** [comment]
I can do it if you want. Photoshop does this stuff really easily.

**Comment by Hugal31** [comment]
@SweptWasTaken feel free to do it if you want.

File: Resources/Prototypes/Entities/Buildings/airlock_types.yml [hunk 1]
```diff
@@ -130,3 +130,110 @@
...
     sprite: Buildings/Doors/airlock_security_glass.rsi
+
+- type: entity
+  parent: Airlock
+  id: AirlockMaint
+  name: Maintenance Hatch
+  components:
+  - type: Sprite
+    sprite: Buildings/Doors/airlock_maint.rsi
+
```
Comment (AJCM-git) [inline Resources/Prototypes/Entities/Buildings/airlock_types.yml:141]: Do not separate components

Comment (Hugal31) [reply Resources/Prototypes/Entities/Buildings/airlock_types.yml:141]: I copy-pasted the previous door prototypes. I'll fix this for the whole file.

Comment (AJCM-git) [inline Resources/Prototypes/Entities/Buildings/airlock_types.yml]: Do not separate components

File: Resources/Prototypes/Entities/Buildings/airlock_types.yml [hunk 2]
```diff
@@ -127,6 +112,103 @@
...
+  - type: Sprite
+    sprite: Buildings/Doors/airlock_maint_command.rsi
+  - type: Icon
+    sprite: Buildings/Doors/airlock_maint_command.rsi
+
+- type: entity
+  parent: Airlock
+  id: AirlockMaintCommon
+  name: Maintenance Hatch
+  suffix: Common
```
Comment (AJCM-git) [inline Resources/Prototypes/Entities/Buildings/airlock_types.yml:154]: I dont thing is a good idea to use suffix in this way, they should be used to show information that the players doesnt need but that is useful for mappers/admins/etc. Players need the feedback of knowing what access they need to enter a door and if you use suffix they wont be able to see it.

Comment (Hugal31) [reply Resources/Prototypes/Entities/Buildings/airlock_types.yml:154]: Why not. I copied the behavior from CEV-Eris, where the maintenance doors are not named IIRC.

Comment (PJB3005) [reply Resources/Prototypes/Entities/Buildings/airlock_types.yml:154]: This is fine.

**Comment by Hugal31** [comment]
I imported the cargo airlock as well. I'll do the blast and secure airlock if I have time before @SweptWasTaken do the security airlocks.

CargoAirlock

**Comment by Hugal31** [comment]
Finally, the security airlocks, thanks to @CrackCocaine!
secairlock

**Pull request merged**